modules/timeline.xql

```diff
@@ -15,10 +15,11 @@ response:set-header("Cache-Control", "no-cache"),
 
 let $start := request:get-parameter("start",1)
 let $count := request:get-parameter("count",10)
-let $timelineURLString := "https://exist-db.org/exist/apps/wiki/modules/feeds.xql?feed=eXist;lts&amp;start=" || $start || "&amp;count=" || $count
-let $data := httpclient:get(xs:anyURI($timelineURLString), false(), ())
-let $entries :=
-    for $entry in $data/httpclient:body//atom:entry
+let $timelineURLString := "https://exist-db.org/exist/apps/wiki/modules/feeds.xql?feed=eXist&amp;start=" || $start || "&amp;count=" || $count
+let $data := http:send-request(
+    <http:request method='get' />,$timelineURLString)
```

Comment:
We could perhaps simply use `fn:doc()`, since the feeds.xql endpoint returns Atom documents - valid XML.

Reply:
@lguariento No need to change this - just an observation.